srcpkgs/aisleriot/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'aisleriot'
 pkgname=aisleriot
 version=3.22.13
-revision=2
+revision=3
 build_style=meson
 # build requires assertions to be turned on -> n_debug=false
 configure_args="-Dtheme_pysol_path=/usr/share/PySolFC/cardsets -Dtheme_pysol=true
  -Dtheme_kde=false -Db_ndebug=false"
-hostmakedepends="desktop-file-utils glib-devel guile intltool itstool pkg-config
+hostmakedepends="desktop-file-utils glib-devel guile3 intltool itstool pkg-config
  pysolfc-cardsets"
-makedepends="guile-devel libcanberra-devel librsvg-devel libatomic_ops-devel"
-depends="guile yelp"
+makedepends="guile3-devel libcanberra-devel librsvg-devel libatomic_ops-devel"
+depends="yelp"
```

Comment:
Is it correct to remove guile from depends?

Reply:
> Why isn't libguile3 a separate package?

Do you mean like in guile2.2?
```sh
$ xbps-query -Rf libguile
/usr/lib/libguile-2.2.so.1.4.2
/usr/share/gdb/auto-load/usr/lib/libguile-2.2.so.1.4.2-gdb.scm
/usr/lib/libguile-2.2.so.1 -> /usr/lib/libguile-2.2.so.1.4.2
```
`libguile` does not make sense without `*.go` and `*.scm` files, which are in the `guile` package:
```
$ xbps-query -Rf guile | grep '\.go'
$ xbps-query -Rf guile | grep '\.scm'
```
`libguile-2.2.so` is just a runtime library, like `libpython3.9.so` in `python3`.